### PR TITLE
Create quality.yml

### PR DESCRIPTION
**Pull request by cefoo**
# Create yml file for quality estimation products

## Type of PR

- Creates the article _quality.yml_

### Checklist:

- [x] I have read the [contributing guidelines](contributing.md).
- [x] I have followed the [style guide](http://machinetranslate.org/style).

### CODE REVIEW

**Comment by bittlingmayer** [comment]
Memsource is missing.

File: _data/quality.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+- id: modelfront
+     name: ModelFront
+     description: Translation quality prediction
+     languages: [ af, sq, am, ar, hy, as, az, ba, eu, be, bn, bs, bg, my, yue, ca, ceb, zh, zh-Hans, zh-Hant, co, hr, cs, da, prs, dv, nl, en, en-gb, en-us, et, fj, fi, fr, fr-ca, fr-fr, fr-ch, fy, gl, ka, de, de-at, de-de, de-ch, el, gu, ht, ha, haw, he, hi, hmn, hu, is, ig, id, iu, ga, it, it-it, it-ch, ja, jv, kn, kk, km, rw, ko, kmr, ckb, ky, lo, la, lv, lt, lmo, nds, lb, mk, mg, ms, ml, mt, mi, mr, yua, mn, ne, no, nb, nn, ny, or, otq, ps, fa, pl, pt, pt-br, pt-pt, pa, ro, ru, sm, gd, sr, sr-Cyrl, sr-Latn, st, sn, sd, si, sk, sl, so, dsb, hsb, es, es-419, es-es, su, sw, sv, tl, ty, tg, ta, tt, te, th, bo, ti, to, tr, tk, uk, ur, ug, uz, uz-Cyrl, uz-Latn, vi, cy, xh, yi, yo, zu ]
```
Comment (bittlingmayer) [inline _data/quality.yml:4]: Maybe put a quote around `no`?

Comment (cefoo) [reply _data/quality.yml]: done

**Comment by cefoo** [comment]
> Memsource is missing.

fixed

**Comment by zouharvi** [comment]
Is this PR waiting for something? Since it's a data-sheet-like page, it's expected to be continually updated (as new companies emerge). We could probably merge.

**Comment by cefoo** [comment]
> Is this PR waiting for something? Since it's a data-sheet-like page, it's expected to be continually updated (as new companies emerge). We could probably merge.

I believe the PR is ready. However, we are thinking of the best way to display this information in an article (or several), as we did with `Engines` and `Languages`.

**Comment by bittlingmayer** [comment]
Let's try to make it like engines.yml, we learnt a few things with that one.

- Make `urls` a list.
- Fix indentation
- id before name
- Add `engine: [ engine id ]` for those that only work with one engine (and in that case, maybe we don't need to include languages?  So we don't need to update the list in 2 places.)

We should also add whether it supports customization or not.  (I believe all do, except for Memsource.  But maybe good to confirm about Kantan.)

**Pull request merged**